hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.test.tsx

```diff
@@ -64,11 +69,20 @@ test("cancel and submit text are NOT being passed into popup modal without crash
             isVisible={true}
             submitHandler={submitFunction}
             cancelHandler={cancelFunction}
+            AnimationPresence={true}
             title={"Test Modal"}
         />
     );
-    expect(getByText("Cancel")).toBeInTheDocument();
-    expect(getByText("Submit")).toBeInTheDocument();
-    fireEvent.click(getByText("Cancel"));
-    fireEvent.click(getByText("Submit"));
+    //New changes to test: passed in the expect() calls inside a waitFor() method, as the modal has a delay between when it is being animated
+    //...and when the words "Cancel" and "Submit" are present on the Modal
+    //So, we must await for this
+    await waitFor(() => {
+        expect(getByText("Cancel")).toBeInTheDocument();
+        expect(getByText("Submit")).toBeInTheDocument();
+    });
+    //Commented out the last 4 lines of this test that were present prior to changes in PopupModal.tsx
```

Comment:
Remove all the unused comments

Reply:
Don't worry, git will handle everything for you. You don't need to worry that the old code won't disappear! Everything in Git is tracked.